src/plugins/DevTools/components/InfoComponent.ts

```diff
@@ -13,4 +38,20 @@ export class InfoComponent extends OwlUIComponent<{}> {
             vNode: vNode,
         });
     }
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    _repr(rangeNode): string {
+        const next = rangeNode.nextSibling;
+        const prev = rangeNode.previousSibling;
+        if (next) {
+            return `BEFORE ${next.id} (${next.value || next.type})`;
+        } else if (prev) {
+            return `AFTER ${prev.id} (${prev.value || prev.type})`;
```

Comment:
OK for me. About the name, how about `nodeRepr` ? It's just a way to represent the node that makes it recognizable for the user: its value if it has one, its type otherwise.